circle/docker-release-image.sh

```diff
@@ -44,14 +61,16 @@ fi
 TAGS_TO_UPDATE+=($IMAGE_TAG)
 
 # Adding rolling tag
-TAGS_TO_UPDATE+=($ROLLING_IMAGE_TAG)
+TAGS_TO_UPDATE+=("${ROLLING_IMAGE_TAG}-${OS_PLATFORM}")
```

Comment:
how will be the rolling tag for the default debian image?

Reply:
Oh, this is a bug, as it will make it have a trailing undesired `-` in that case. I will change it, thanks!